src/Microsoft.ML.StandardLearners/Standard/Simple/SimpleTrainers.cs

```diff
@@ -216,6 +254,28 @@ public PriorTrainer(IHostEnvironment env, Arguments args)
             Host.CheckValue(args, nameof(args));
         }
 
+        /// <summary>
+        /// Initializes PriorTrainer object.
+        /// </summary>
+        public PriorTrainer(IHost host, SchemaShape.Column label, SchemaShape.Column weight)
```

Comment:
>SchemaShape.Column label [](start = 40, length = 24)

The constructor signature should be 
`PriorTrainer(IHostEnvironment env, string labelColumn, string weightColumn = null)`

Reply:
Ok, I made those Strings instead of SchemShape.Column

---
In reply to: [218244513](https://github.com/dotnet/machinelearning/pull/876#discussion_r218244513) [](ancestors = 218244513)